src/main/resources/jaxws/bindings.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jaxws:bindings xmlns:jaxws="http://java.sun.com/xml/ns/jaxws" xmlns:jaxb="http://java.sun.com/xml/ns/jaxb" xmlns:xjc="http://java.sun.com/xml/ns/jaxb/xjc" xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" xmlns:xs="http://www.w3.org/2001/XMLSchema" version="2.0">
+	<jaxws:package name="com.sentrysoftware.matsya.winrm.service"/>
+	<jaxws:enableWrapperStyle>false</jaxws:enableWrapperStyle>
+	<jaxws:enableAsyncMapping>false</jaxws:enableAsyncMapping>
+	<jaxws:bindings schemaLocation="../xsd/dsp8033_1.0.xsd" node="wsdl:definitions/wsdl:types/xs:schema[@targetNamespace='http://schemas.xmlsoap.org/ws/2004/09/enumeration']">
+		<jaxb:globalBindings xmlns:jaxb="http://java.sun.com/xml/ns/jaxb" xmlns:xs="http://www.w3.org/2001/XMLSchema"/>
+		<jaxb:schemaBindings>
+			<jaxb:package name="com.sentrysoftware.matsya.winrm.service.enumeration"/>

```

Comment:
The packages are incorrect in this bindings file. Please make sure everything is consistent in `WinRM.wsdl` and the `*.xsd` files.